CMakeLists.txt

```diff
@@ -68,18 +77,34 @@ set(BIPP_GPU "OFF" CACHE STRING "GPU backend")
 set_property(CACHE BIPP_GPU PROPERTY STRINGS
 	"OFF" "CUDA" "ROCM"
 	)
-set(BUILD_SHARED_LIBS "ON" CACHE STRING "Build as shared library") # default to shared
-set(BIPP_INSTALL "LIB" CACHE STRING "Install the librariy (LIB), the python module (PYTHON) or nothing (OFF). For use with pip install (PIP) is also available.")
-set_property(CACHE BIPP_INSTALL PROPERTY STRINGS
-	"LIB" "PYTHON" "PIP" "OFF"
-	)
 set(CMAKE_HIP_ARCHITECTURES "gfx803;gfx900;gfx906" CACHE STRING "HIP GPU architectures")
 
 set(BIPP_BUILD_TYPE "OFF" CACHE STRING "If set, overrides the CMAKE_BUILD_TYPE variable.")
 set_property(CACHE BIPP_BUILD_TYPE PROPERTY STRINGS
 	"Debug" "Release" "MinSizeRel" "RelWithDebInfo")
 if(BIPP_BUILD_TYPE)
-	set(CMAKE_BUILD_TYPE ${BIPP_BUILD_TYPE} CACHE STRING "Build type" FORCE)
+	set(CMAKE_BUILD_TYPE ${BIPP_BUILD_TYPE})
+endif()
+
+
+# Python install location
+if(BIPP_PYTHON)
+	set(Python_FIND_FRAMEWORK LAST) # Prefer Brew/Conda to Apple framework python
+	find_package(Python 3.6 COMPONENTS Interpreter Development REQUIRED)
```

Comment:
Is it going to look specifically for Python 3.6?

Reply:
That's just the minimum version requirement. It's unchanged from where it was moved from in python/CMakeLists.txt.